app/assets/stylesheets/application.scss

```diff
@@ -1,44 +1,68 @@
 @import "bootstrap";
 
+$orange: #f2580b;
+@mixin orange_background {
+  background-color: $orange;
+  border-color: $orange;
+}
+body {
+  background-color: #eee;
+  font-family: 'Roboto', sans-serif;
+}
+
+.navbar-header {
+  border-bottom: 1px solid gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.